cookbooks/chef-client/attributes/default.rb

```diff
@@ -48,20 +48,25 @@
 
 # Configuration for chef-client::cron recipe.
 default['chef_client']['cron'] = {
-  'minute' => '0',
-  'hour' => '0,4,8,12,16,20',
+  'minute' => '0,30',
+  'hour' => '*',
```

Comment:
lets make the cron run every four hour once following earlier config. current one can make release delay

Reply:
These changes have been made! @shudarshon-binary